README.md

```diff
@@ -184,6 +184,14 @@ git checkout stdlib-fpm
 fpm build --profile release
 ```
 
+**Alternative**: as fpm does not currently support fypp natively, building stdlib is done in two steps (a) launch the preprocessor through the fpm-deployment.sh script, which creates a subfolder `stdlib-fpm` (b) build the project using the processed files within the latter subfolder. Which can be done with the following commands:
+
+```sh
+source ./ci/fpm-deployment.sh
```

Comment:
```suggestion
git checkout stdlib
source ./ci/fpm-deployment.sh
```

The note sounds good to me. 

If @awvwgk (and others) is/are ok, it is good for me

Reply:
Question: is the "git checkout stdlib" command necessary here? So far I have not needed this line. Here my working env:
Windows+WSL+VSCode
So, with VS Code and git installed locally, I can track changes and swap branches. I use the WSL for launching the build script, and also compiling under Linux with the compilers installed in the WSL. Then I can use the same folder to recompile with my Windows compilers.

Reply:
Yes, you are right. 
I meant `git clone stdlib` or `git checkout master`. 
But it might be to specific to my environment. So you can ignore this comment. Sorry for the  noise.

Reply:
I see, I kind of assumed that whomever will use this, would have already done `git clone stdlib`. No problem about it